### PR TITLE
Proposed layout for blog post

### PR DESCRIPTION
**Pull request by IshaDakota**
#345 

Went down sort of a wormhole of playing with layout to achieve a shorter line length for blog posts. This a merely a proposal. Consider at your leisure and discretion (whoever considers such proposals). :) 

cross-site_scripting_attack_detection

### CODE REVIEW

File: behat/features/bootstrap/FeatureContext.php [hunk 1]
```diff
@@ -26,6 +26,6 @@ public function iShouldTheTextUnderTheMainContent($text) {
    * @Then I should see the author :author
    */
   public function iShouldSeeTheAuthor($author) {
-    $this->assertElementContains('.date', $author);
+    $this->assertElementContains('.post-author', $author);
```
Comment (amitaibu) [inline behat/features/bootstrap/FeatureContext.php]: @IshaDakota oh my, it even includes a Behat fix! :)

Comment (IshaDakota) [inline behat/features/bootstrap/FeatureContext.php]: @amitaibu well, I broke like the only test. It's the least i could do ;)

**Comment by IshaDakota** [comment]
fireshot capture 40 - cross-site scripting attack detecti_ - http___0 0 0 0_9000_content_xss-attack_

**Comment by IshaDakota** [comment]
@RachelBaram for review. This addresses #329 (in a better way IMO) and #419 

Re typography, a few of notes:
- on first scan, the body font is going to look small to you.It is, however a much more appropriate size for this sans-serif font. It's readable at this size and lines look good. If we are committed to a larger body font, I'd suggest finding a new font, probably something with a serif.
- I think it's best to tackle typography page by page, and then go back and fix the global variables. That way we won't accidentally break format on some pages.
-Capitalizing the h1 rather than uppercase makes it less screamy at these sizes. We should apply throughout.

Also, I removed the bix shadow from the images - looks better IMO

Played a little with the pagers as well.

There may be some work needed for mobile.

**Comment by RachelBaram** [comment]
I like that you narrowed the text further and added the face and other elements to the sides. I trust your typography decisions since you are a stickler there. I can not see it well from the screenshot to give any worthwhile input. I have not yet figured it out but the blog title seems off. It looks left heavy. Maybe it should start at the the same margin as the blog text?

**Comment by IshaDakota** [comment]
@RachelBaram I think the lack of balance is due to you looking at the whole page image at once. You would never see it that way on a screen. See below a 13-inch macbook.

Also (something I meant to do before) a small icon in the left column let's the viewer know they are at a blog post (if they came from the outside), and gives some weight to that side.

... and also, I changed the title to the Raleway font, which I think was the intended font for headers, and looks super nice against the Hind font used in the body.

fireshot capture 43 - migration best practi_ - http___0 0 0 0_9000_content_migration-best-practices_

**Comment by RachelBaram** [comment]
@IshaDakota  I think it is the author name under the title that is throwing off the balance. Not sure of the solution since it is important that it be close to the title. Everything else looks really great. I love the font changes and the box adds a richness.

**Comment by amitaibu** [comment]
Closing this on in favor of #433

**Pull request closed without merging**